.github/actions/setup-build/action.yml

```diff
@@ -39,3 +39,18 @@ runs:
     with:
       key: ${{ runner.os }}-stablehlo_build_assets-${{ inputs.llvm-version }}
       max-size: 4G
+
+  # Install Python/Numpy for API tests
+  - name: Install Python and Pip
+    uses: actions/setup-python@v4
+    with:
+      python-version: '3.10.6'
+      cache: 'pip' # caching pip dependencies
+  - name: Install MLIR python requirements
+    shell: bash
+    run: |
+      pwd
+      ls
+      ls llvm-project
+      ls $GITHUB_WORKSPACE/llvm-project
```

Comment:
Why do we need these commands?

Reply:
Ah. Debug code. Was hitting an issue where it couldn't find `stablehlo/llvm-project/` directory, and I added these to figure out what was going on. Turns out if you pass an environment variable as an input argument, it escapes the string. Was creating `stablehlo/\$GITHUB_WORKSPACE/\$LLVM_PROJ_DIR/llvm-project` instead.

Will remove.